usaspending_api/awards/v2/filters/view_selector.py

```diff
@@ -1,13 +1,7 @@
-from usaspending_api.awards.models_matviews import SummaryCfdaNumbersView
-from usaspending_api.awards.models_matviews import SummaryNaicsCodesView
-from usaspending_api.awards.models_matviews import SummaryPscCodesView
-from usaspending_api.awards.models_matviews import SummaryAwardView
-from usaspending_api.awards.models_matviews import SummaryTransactionGeoView
-from usaspending_api.awards.models_matviews import SummaryTransactionMonthView
-from usaspending_api.awards.models_matviews import SummaryTransactionView
-from usaspending_api.awards.models_matviews import SummaryView
-from usaspending_api.awards.models_matviews import UniversalAwardView
-from usaspending_api.awards.models_matviews import UniversalTransactionView
+from usaspending_api.awards.models_matviews import (SummaryCfdaNumbersView, SummaryNaicsCodesView, SummaryPscCodesView,
```

Comment:
Seems a little unnecessary to change the imports without other edits in the file.
They were originally conforming the [Google's Python style guide](https://google.github.io/styleguide/pyguide.html#Imports_formatting)

Reply:
That's fair. I initially made changes to the file (which included adding imports) but ended up removing them.